src/main/java/duke/task/Deadlines.java

```diff
@@ -0,0 +1,47 @@
+package duke.task;
+
+import duke.Parser;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
```

Comment:
Shouldn't this class be in singular form since it is only referring to one deadline? 😄

Reply:
Yep I agree. The class may be misunderstood as a collection of Task objects, so it might be better to use the singular form.

Reply:
oh yea that is true, thanks for pointing this out!